packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -105,6 +105,11 @@ export interface ITestObjectProvider {
 	 * containerRuntime/dataRuntime used in fluidEntryPoint will be used as is from what is passed in.
 	 *
 	 * @param packageEntries - list of code details and fluidEntryPoint pairs.
+	 *
+	 * @remarks
+	 *
+	 * The containers returned by the loader are not guaranteed to be in Connected state.
```

Comment:
i don't really like this call out on the loaders, as its not totally true as it depends on which load options the user of the loader specifies

Reply:
Hmm fair. Doc-ing this is tricky, I'll just revert that part for now.